src/store/audio.ts

```diff
@@ -1473,41 +1494,42 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
       async (
         { state, getters, actions },
         {
-          audioKeys,
+          termConfirmedAudioKeys,
           dirPath,
           callback,
         }: {
-          audioKeys: AudioKey[];
+          termConfirmedAudioKeys: TermConfirmedAudioKey[];
           dirPath?: string;
           callback?: (finishedCount: number) => void;
         },
       ) => {
         if (state.savingSetting.fixedExportEnabled) {
           dirPath = state.savingSetting.fixedExportDir;
         } else {
-          dirPath ??= await window.backend.showSaveDirectoryDialog({
+          const selectedDirPath = await window.backend.showSaveDirectoryDialog({
             title: "音声を保存",
           });
+          if (selectedDirPath == undefined) {
+            return "canceled";
+          }
+          dirPath = selectedDirPath;
         }
-        if (dirPath) {
-          const _dirPath = dirPath;
 
-          let finishedCount = 0;
+        let finishedCount = 0;
 
-          const promises = audioKeys.map((audioKey) => {
-            const name = getters.DEFAULT_AUDIO_FILE_NAME(audioKey);
-            return actions
-              .GENERATE_AND_SAVE_AUDIO({
-                audioKey,
-                filePath: path.join(_dirPath, name),
-              })
-              .then((value) => {
-                callback?.(++finishedCount);
-                return value;
-              });
-          });
-          return Promise.all(promises);
-        }
+        const promises = termConfirmedAudioKeys.map((termConfirmedAudioKey) => {
+          const name = getters.DEFAULT_AUDIO_FILE_NAME(termConfirmedAudioKey);
+          return actions
+            .GENERATE_AND_SAVE_AUDIO({
+              termConfirmedAudioKey,
+              filePath: path.join(dirPath, name),
+            })
+            .then((value) => {
+              callback?.(++finishedCount);
+              return value;
+            });
+        });
+        return Promise.all(promises);
       },
```

Comment:
変更が含まれてるとこにもsuggestコメントできるかテスト
```suggestion

aaa
},
```

Reply:
できてそう！！！実際applyしたらエラーになるとかもありえるかもだけど！！！